index.html

```diff
@@ -50,10 +50,10 @@
 <!--                    <a class="st-root-link item-shop st-has-dropdown nav-link" href="shop/" data-dropdown="shop">Shop</a>-->
                 </li>
                 <li class="st-nav-section st-nav-secondary nav-item">
-                    <a class="btn btn-rounded btn-outline me-3 px-3" href="https://bambai.risingacademies.com/" target="_blank">
+                    <a class="btn btn-rounded btn-outline me-3 px-3" href="https://bambai.risingacademies.com/auth-login" target="_blank">

```

Comment:
Did we have to change this? I'd think it will default to login if the user isn't logged on, or will go to the landing page. @GeorgePadmore can you verify this please

Reply:
Yes that's correct. It would have default to login if the user isn't logged on.
But I thought instead of doing that round trip, it goes directly to the login page.
I can revert that change if its not necessary
@surabhi013

Reply:
What happens if the user is logged in? If this line is changed, does it go to the landing page for logged in user, or to the login page?

Reply:
If the user is logged in, the login page won't redirect the user to the home page. You are correct. I have changed that route to "https://bambai.risingacademies.com/". That solves it all.
Thanks